pkg/client/mount.go

```diff
@@ -224,18 +226,28 @@ func readlink(ctxt context.Context, device string) (string, error) {
 }
 
 func deviceFromMount(ctxt context.Context, file string) (string, error) {
-	cmd := []string{"sh", "-c", fmt.Sprintf(`"grep '%s' /proc/mounts | awk '{print \$1}'"`, file)}
-	out, err := co.RunCmd(ctxt, cmd...)
-	if err != nil {
-		return "", err
-	}
-	dev, err := readlink(ctxt, out)
+
+        // Multiple Unix pipes does not work well in Golang
```

Comment:
check alignment, off by 2

Reply:
Same as before. GitHub shows the diff'ed code off by 2 spaces but in the code file all function code are indented by 8 spaces.

Reply:
Fixed.